tests/cycle/test_cycle.py

```diff
@@ -143,7 +143,7 @@ def test_get_cycles_starttime_tag(get_client):
 
     df = get_client.get_cycles(**query)
 
-    assert df.shape == (NUM_ROWS, len(select_columns))
+    assert df.shape != (NUM_ROWS, len(select_columns))
```

Comment:
It there a more posiive criteria we can verify at at this point?  The result meets our expectations.  This is more a verify the result is not a mineral.  Should number of rows beccome EXPECTED_NUM_ROWS?